### DISCUSSION TITLE
XLIFF support

### DISCUSSION THREAD

**Discussion by berserkwarwolf** [tolgee/tolgee-platform#1081] [Q&A] [answered]
Hi, I've been talking about using Tolgee in my company, but everyone kept bringing up that translators prefer Trados to anything else. So, I was thinking about using XLIFF as a way to let them use whatever they want, and then bring back the XLIFF into Tolgee for review & collaboration.

I've seen that XLIFF is not currently implemented or being worked on, but it is in the roadmap. Is there an estimated / projected timeframe when work will begin on this, is there an issue # where I could follow it's progress ?

**Comment by JanCizmar** [comment, marked as answer]
Hi there!

You can already export to XLIFF and import to XLIFF in Tolgee Platform, so this shouldn't be a problem.
